src/bitcoin.ts

```diff
@@ -155,13 +157,13 @@ const port = process.env.SIDETREE_BITCOIN_PORT || config.port;
 let server: any;
 let blockchainService: SidetreeBitcoinProcessor;
 try {
-  blockchainService = new SidetreeBitcoinProcessor(config, ionBitcoinVersions);
+  blockchainService = new SidetreeBitcoinProcessor(config, );
```

Comment:
This doesn't look right

Reply:
Will fix in the actual version hike PR next.